devices/homeetec.js

```diff
@@ -0,0 +1,360 @@
+        const tuyaDatapoint = tuyaDatapoints.find((d) => d[0] === dp);
+
+        const result = {};
+            
+        const key = tuyaDatapoint ? tuyaDatapoint[1] : Object.keys(COVER_DP).find((d) => COVER_DP[d] === dp);
+
+        if (DP_OPTIONS[dp] || false) {
+            const options = DP_OPTIONS[dp];
+            const selectedOption = Object.keys(options).find((d) => options[d].toString() === value.toString());
+
+            if (selectedOption || false) {
+                result[key] = selectedOption;
+                meta.logger.debug(`homeetec_cover (FZ) DEBUG options, dp: ${dp}, key: ${key}, value: ${value}, options: ${JSON.stringify(options)}, dpValue: ${JSON.stringify(dpValue)}, result: ${JSON.stringify(result)}`);
+            } else {
+                meta.logger.debug(`homeetec_cover (FZ) Invalid value '${value}' provided for DP#${dp}, options: ${JSON.stringify(options)}`);
+            }
+
+        } else if (key === DP_KEYS.Position) {
+            const invert = tuya.isCoverInverted(meta.device.manufacturerName) ? !options.invert_cover : options.invert_cover;
+            const position = invert ? 100 - (value & 0xff) : value & 0xff;
+            result[key] = position;
+            meta.logger.debug(`homeetec_cover (FZ) DEBUG position, dp: ${dp}, key: ${key}, value: ${value}, dpValue: ${JSON.stringify(dpValue)}, result: ${JSON.stringify(result)}`);
+            
+        } else if ([DP_KEYS.Backlight, DP_KEYS.ChildLock, DP_KEYS.Switch, DP_KEYS.Switch1, DP_KEYS.Switch2].includes(key)) {
+            result[key] = getOnState(value);
+            meta.logger.debug(`homeetec_cover (FZ) DEBUG boolean, dp: ${dp}, key: ${key}, value: ${value}, dpValue: ${JSON.stringify(dpValue)}, result: ${JSON.stringify(result)}`);
+        }
+        
+        if (Object.keys(result).length > 0) {
+            return result;
+        }
+    },
+};
+
+const getDefintions = (model = null) => {
+    const definitions = [
+        {
+            fingerprint: [{
+                modelID: 'TS0001',
+                manufacturerName: '_TZ3000_bmqxalil',
+            }],
+            model: 'TS0001_switch_1_gang',
+            vendor: 'TuYa',
+            description: '1-Gang switch with backlight',
+            fromZigbee: [fz.on_off, tuya.fz.power_on_behavior_2, tuya.fz.backlight_mode_off_on],
+            toZigbee: [tz.on_off, tuya.tz.power_on_behavior_2, tuya.tz.backlight_indicator_mode_2],
+            exposes: [
+                e.switch(),
+                e.power_on_behavior(),
+                exposes.binary('backlight_mode', ea.STATE_SET, 'on', 'off').withDescription('Backlight mode'),
+            ],
+            configure: async (device, coordinatorEndpoint, logger) => {
+                await tuya.configureMagicPacket(device, coordinatorEndpoint, logger);
+                await reporting.bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
+            },
+        },
+        {
+            fingerprint: [{
```

Comment:
Could you check if this works with: 

```js
{
    fingerprint: tuya.fingerprint('TS0002', ['_TZ3000_in5qxhtt']),
    model: 'TS0002_switch_2_gang',
    vendor: 'TuYa',
    description: '2 gang switch',
    extend: tuya.extend.switch({powerOnBehavior2: true, backlightModeOffOn: true, endpoints: ['l1', 'l2']}),
    endpoint: (device) => {
        return {'l1': 1, 'l2': 2};
    },
    meta: {multiEndpoint: true, disableDefaultResponse: false},
    configure: async (device, coordinatorEndpoint, logger) => {
        await tuya.configureMagicPacket(device, coordinatorEndpoint, logger);
        await reporting.bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
        await reporting.bind(device.getEndpoint(2), coordinatorEndpoint, ['genOnOff']);
    },
}
```

Reply:
yes, that's extactly the code I added in homeetec.js,
once it's approved by you, I will move to tuya.js (wanted to avoid conflicts)